src/modules/.internal/utils/getLocale.ts

```diff
@@ -1,6 +1,6 @@
 export const getLocale = (): string => {
   // Check if running in a non-browser environment (e.g., Node.js or older browsers).
-  if (navigator === undefined) {
+  if (typeof navigator !== undefined) {
```

Comment:
IMO `typeof navigator === 'undefined'` should be the condition here. 

Your condition might seem logical at first glance, but it has a critical flaw.

The `typeof` operator returns a string indicating the type of the operand. Therefore, if you use `typeof navigator`, it will either return the string 'object' (in a browser) or 'undefined' (in a non-browser environment like Node.js). Using `typeof navigator !== undefined` would always evaluate to true, regardless of whether navigator exists or not.

Reply:
Make sense.
Thanks for correcting me.

Reply:
Hey @sakigo9 ,  Could you make the suggested changes ?

Reply:
Hi @tarun-khanna 
I have made the necessary changes.